plugin/front/ticket.form.php

```diff
@@ -32,15 +32,17 @@
         Html::back();
     }
 
-    $ticket->updateItem($ticketId);
+    $ticket->updateItem($ticketId, $post);
 
     Html::redirect($ticket->getFormURL() . '?mode=' . PluginIserviceTicket::MODE_CLOSE . '&id=' . $ticketId);
 } elseif (!empty($update)) {
     $ticket->check($id, UPDATE, $post);
 
+    $post = $ticket->preProcessPostData($post);
```

Comment:
When there is no need to have an instance, the best practice is to declare the function as static.
```suggestion
    $post = PluginIserviceTicket::preProcessPostData($post);
```